### PR TITLE
🎨 New UI

### PR DESCRIPTION
**Pull request by CryogenicPlanet**

### CODE REVIEW

File: src/components/dropzone/dropzone.tsx [hunk 1]
```diff
@@ -110,6 +110,13 @@ const Dropzone = () => {
     }
   }, [globalReset])
 
+  useEffect(() => {
+    setDropzoneRef(dropzoneRef)
+    return () => {
+      delete FileStore.dropzoneRef
+    }
+  }, [])
+
```
Comment (CryogenicPlanet) [inline src/components/dropzone/dropzone.tsx:119]: Now dropzone ref can be accessed globally and opened with `FileStore.openFileDrawer`

File: src/store/userStore.ts [hunk 2]
```diff
@@ -144,5 +148,26 @@ class UserStore extends AbstractStore {
...
+  }, [localUsageCounter, isActiveUser])
+
+  useEffect(() => {
+    if (disabled === true) {
+      setDisplay(false)
+    }
+  }, [disabled])
+
+  return displayable
+}
```
Comment (CryogenicPlanet) [inline src/store/userStore.ts:173]: Custom hook to easily determine if the banner is open or not statefully

File: src/store/fileStore.ts [hunk 3]
```diff
@@ -108,6 +111,19 @@ class FileStore extends AbstractStore {
...
+  }
+
+  @action('Open file drawer')
+  openFileDrawer = () => {
+    const { dropzoneRef } = this
+    if (dropzoneRef && dropzoneRef.current) {
+      dropzoneRef.current.click()
+    }
+  }
+
```
Comment (CryogenicPlanet) [inline src/store/fileStore.ts:126]: Implementation of `Filestore.openFileDrawer`

**Review by gaurangbharti1 — APPROVED** [review]
LGTM!

**Pull request merged**